docs/getting_started.md

```diff
@@ -9,7 +9,7 @@ Follow this guide to:
 ## Pre-requisites
 
 First download the Arm GNU toolchain `arm-none-eabi` 14.2.Rel1 from [here](https://developer.arm.com/downloads/-/arm-gnu-toolchain-downloads).
-Make sure to make it available on your path `PATH` by checking GCC version:
+Make sure to make it available on your path `PATH` and then check GCC version is reported correctly:
 
 ```shell
 $ arm-none-eabi-gcc --version
```

Comment:
`-v` makes it more obvious if you have the right version, the homebrew and ARM ones can look near identical with `--version` but `-v` shows: `gcc version 14.2.1 20241119 (Arm GNU Toolchain 14.2.Rel1 (Build arm-14.52))`

Reply:
will update this

Reply:
I actually find `-v` a bit more confusing... `Arm GNU Toolchain 14.2.Rel1 ` is shown on both.

Reply:
let's discuss this in a future PR, I was atually updating macos stuff here :-)